dll/shellext/sendmail/lang/ro-RO.rc

```diff
@@ -0,0 +1,7 @@
+LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+    IDS_DESKLINK, "Desktop (Creare comandă rapidă)"
```

Comment:
Elsewhere in the codebase, „Desktop” is translated as „Birou”. Either translate it here too, or consider updating the rest of the translations.